src/app/landing/mentors/mentors.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Inject, Injectable } from "@angular/core";
+import { RepositoryService } from "../shared/services/repository.service";
+import { Mentor, MentorsRepository } from "../shared/interfaces/mentor";
+import { Observable } from "rxjs";
+
+
+@Injectable({
+    providedIn: 'root'
+})
+export class MentorsService{
+    constructor(@Inject(RepositoryService) private repo: MentorsRepository){}
```

Comment:
Why not just?
```
constructor(private repo: RepositoryService){}
```

Reply:
I needed to inject service with more specific interface then RepositoryService, so that this class didn't depend on interface that is don't use.